optimum/intel/openvino/configuration.py

```diff
@@ -816,11 +822,20 @@ def post_init(self):
         if self.dataset is not None:
             speech_to_text_datasets = list(PREDEFINED_SPEECH_TO_TEXT_DATASETS.keys())
             stable_diffusion_datasets = list(PREDEFINED_SD_DATASETS.keys())
-            if self.dataset not in LANGUAGE_DATASETS + speech_to_text_datasets + stable_diffusion_datasets:
+            language_datasets = list(LANGUAGE_DATASETS.keys())
+            if (
+                self.dataset
+                not in language_datasets
+                + CAUSAL_LANGUAGE_DATASETS
+                + speech_to_text_datasets
+                + stable_diffusion_datasets
+            ):
```

Comment:
not very important but the resulting list will have some redundant values

Reply:
Converted to sets.